### PR TITLE
feat[plugin]: add labels to cluster nodes

### PR DESCRIPTION
**Pull request by agreewangshuai**

### CODE REVIEW

File: plugin/lables.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+package plugin
```
Comment (allencloud) [inline plugin/lables.go:1]: You should update this file name to be labels.go rather than lables.go

File: plugin/lables.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+	"github.com/alibaba/sealer/client"
+	"github.com/alibaba/sealer/logger"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+type LabelsNodes struct {
+	clusterName string
+	data        map[string][]label
```
Comment (allencloud) [inline plugin/lables.go:14]: What is this attribute `data` for? Could you add more description for it?

File: plugin/lables.go (hunk 1 above)
Comment (agreewangshuai) [reply plugin/lables.go:1]: ok

**Review by agreewangshuai — COMMENTED** [review]
labels plugin in Clusterfile:
---
apiVersion: sealer.aliyun.com/v1alpha1
kind: Plugin
metadata:
  name: LABEL
spec:
  data: |
     192.168.0.2 ssd=true
     192.168.0.3 ssd=true
     192.168.0.4 ssd=true
     192.168.0.5 ssd=false,hdd=true
     192.168.0.6 ssd=false,hdd=true
     192.168.0.7 ssd=false,hdd=true
---
LabelsNodes.data key = ip
[]lable{{key=ssd,value=false}, {key=hdd,value=true}}

File: plugin/labels.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	value string
+}
+
+func NewLabelsNodes() Interface {
+	return &LabelsNodes{
+		data: map[string][]label{},
+	}
+}
+
+func (l LabelsNodes) Run(context Context, phase Phase) {
```
Comment (fanux) [inline plugin/labels.go:46]: return error

all the logger -> return err

**Review by fanux — APPROVED** [review]

**Pull request merged**